<!--xml-->
<launch>
    <include file="$(find monoCam)/launch/monoCam.launch" />
    <node pkg="teamb_ui" type="teamb_ui" name="UI" output="screen" />
    <node pkg="OpenTLD" type="destinationTLD" name="OpenTLD" output="screen" />
    <node pkg="eod_nav" type="eod_nav.py" name="EOD_Navigation" output="screen"/>
    <node pkg="rosserial_python" type="serial_node.py" name="Motor_Ctrl" args="/dev/ttyACM0" />
</launch>
